### DISCUSSION TITLE
Broken Pages

### DISCUSSION THREAD

**Discussion by RyanLH95** [community/community#63709] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I am having an issue with my website. As I have just made it live, however only the main page opens but when I click 'quiz', i get a 404 error page the actual quiz page does not work either. Can someone help please? 

https://ryanlh95.github.io/langwhich/

**Comment by radar-blip** [comment]
Works as it should.
Seems you have fixed the problem already. 👍🏻

**Reply by RyanLH95** [reply]
Yeah I managed to fix it in the end. The href URL in the <a> tags had the full directory as the url. It looks like GitHub doesn't like that. Thank you though.

**Comment by EnriquezAlberto** [comment]
You were able to solve it, right? because i see it well

**Reply by RyanLH95** [reply]
Yeah I managed to fix it in the end. The href URL in the <a> tags had the full directory as the url. It looks like GitHub doesn't like that. Thank you though.

**Comment by mehdi-najaran** [comment]
Apparently the problem has been solved 🧑‍💻😍
Tell me if I can help?

**Reply by RyanLH95** [reply]
Yeah I managed to fix it in the end. The href URL in the <a> tags had the full directory as the url. It looks like GitHub doesn't like that. Thank you though.
